### ISSUE TITLE
Feedback from testing:  Provide an `adsl_metabolic` rather than a `dm_metabolic`.

### ISSUE THREAD

**Issue by manciniedoardo** [pharmaverse/admiralmetabolic#73]
### Background Information

If we provide an `adsl_metabolic` dataset instead of `dm_metabolic`, then the code in the "Read in Data" section of the vignettes could be simplified.

### Definition of Done

- [ ] Evaluate if we want to make this change
- [ ] Make change or close issue

**Comment by manciniedoardo** [comment]
@pharmaverse/admiralmetabolic thoughts?

**Comment by starosto** [comment]
@manciniedoardo did you mean `admiral::admiral_adsl` as a basis? or where to find `adsl_metabolic`?
Personally, I would prefer to use `ADSL` as a population base instead of `DM`

**Comment by AndersAskeland** [comment]
I agree that simplifying the read in sections would be great, and can't see (right now at least) any reason not to change.

I am somewhat busy right now, but if we agree to change it, I could try making the change over the weekend.

**Comment by Siddhesh2097** [comment]
So we need to create adsl_metabolic data? I agree to simplify the read in sections. But I'm also busy with office work right now as I'm back from long vacation atleast till next week.

**Comment by manciniedoardo** [comment]
Agree - order of operations should be:
- Move existing test sdtms to pharmaversesdtm (#95)
- Add LB dataset directly to pharmaversesdtm (#94)
- Implement admiralmetabolic_adsl and simplify read in data sections (#73)
- All other work on ADLB tests/analysis (#96 #92 etc)

**Comment by AndersAskeland** [comment]
@manciniedoardo @Siddhesh2097  In relation to the order of operations. Would it be better to do https://github.com/pharmaverse/admiralmetabolic/issues/73 before moving data over to pharmaversesdtm?

**Comment by manciniedoardo** [comment]
I think we'll still need dm_metabolic in pharmaversesdtm, and this task is just saying to provide adsl_metabolic in admiralmetabolic. So I don't think it matters so much which order we do this

**Comment by AndersAskeland** [comment]
If we want to calculate HSI, we need diabetes status. While I would expect this to be usually found in MH, would be easier to us to use DIABFL or something similar in ADSL.

**Comment by AndersAskeland** [comment]
Hey @Keita-Takahashi00 ! I am still not able to add you, but feel free to assign yourself if possible.

**Comment by AndersAskeland** [comment]
@Keita-Takahashi00 Your assigned now. Let me know if you need some assistance in getting started 👍

**Comment by Keita-Takahashi00** [comment]
@AndersAskeland Thank you so much! I If I run into any difficulty, I’ll chat you. (^o^)/

**Comment by AndersAskeland** [comment]
I am actually thinking that the easiest solution for creating an `metabolic_adsl` could be to read directly from `admiral::adsl` and filter data in a similar way as we did in `metabolic_dm`.
